sailor/pai/alerts.py

```diff
@@ -0,0 +1,129 @@
+"""
+Retrieve Alert information from the alert re-use service.
+
+Classes are provided for individual Alert as well as groups of Alerts (AlertSet).
+"""
+
+
+from .constants import ALERTS_READ_PATH
+from .utils import PredictiveAssetInsightsEntity, _pai_application_url
+from ..assetcentral.utils import _fetch_data, _add_properties, _parse_filter_parameters, ResultSet
+from ..utils.timestamps import _odata_to_timestamp_parser
+
+
+@_add_properties
+class Alert(PredictiveAssetInsightsEntity):
+    """PredictiveAssetInsights Alert Object."""
+
+    # Properties (in PredictiveAssetInsights terminology) are:
+    # AlertId, AlertType, AlertTypeDescription, Category, ChangedBy, ChangedOn, Count, CountryID,
+    # CreatedBy, CreatedOn, CustomProperty, Description, EquipmentDescription, EquipmentID, EquipmentName,
+    # ErrorCodeDescription, ErrorCodeID, FunctionalLocationID, FunctionalLocationName, FunctionalLocationDescription,
+    # IndicatorDescription, IndicatorGroupID, IndicatorGroupName, IndicatorID, IndicatorName, LastOccuredOn,
+    # MaintenancePlant, ModelDescription, ModelID, ModelName, NotificationId, OperatorID, OperatorName,
+    # lannerGroup, PlanningPlant, Processor, RefAlertTypeId, SerialNumber, SeverityCode, Source, StatusCode,
+    # TemplateID, TemplateName, TopEquipmentDescription, TopEquipmentID, TopEquipmentName, TopFunctionalLocationID,
+    # TopFunctionalLocationName, TopFunctionalLocationDescription, TriggeredOn
+
+    @classmethod
+    def get_property_mapping(cls):
+        """Return a mapping from PredictiveAssetInsights (PAI) terminology to our terminology."""
+        return {
+            'id': ('AlertId', None, None, None),
```

Comment:
get_property_mapping does not work anymore and we actually don't have a test that shows that. I won't add a test for this because the test should be obsolete after the "big refactoring"

the function needs to be named `_get_legacy_mapping`

Reply:
done